sgx-types/src/report.rs

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! Section 38.15
+//! The REPORT structure is the output of the EREPORT instruction, and must be 512-Byte aligned.
+
+use super::{attr::Attributes, isv, misc::MiscSelect};
+use testing::testaso;
+
+/// This struct is separated out from the Report to be usable by the Quote struct.
+/// Table 38-21
+#[derive(Default)]
+#[repr(C)]
+pub struct ReportBody {
+    /// The security version number of the processor
+    pub cpusvn: u128,
+    /// Bit vector specifying which extended features are saved to the
+    /// MISC region of the SSA frame when an AEX occurs
+    pub miscselect: MiscSelect,
+    reserved0: [u32; 7],
+    /// Attributes of the enclave (Section 38.7.1)
+    pub attributes: Attributes,
+    /// Value of SECS.MRENCLAVE
+    pub mrenclave: [u8; 32],
+    reserved1: [u32; 8],
+    /// Value from SECS.MRSIGNER
+    pub mrsigner: [u8; 32],
+    reserved2: [u32; 24],
+    /// Product ID of the enclave
+    pub isvprodid: isv::ProdId,
+    /// Security version number of the enclave
+    pub isvsvn: isv::Svn,
+    reserved3: [u32; 15],
+    /// Data provided by the user and protected by the Report's MAC (Section 38.15.1)
+    pub reportdata: [u16; 32],
+}
+
+/// Table 38-21
+#[derive(Default)]
+#[repr(C, align(512))]
+pub struct Report {
+    /// The body of the Report
+    reportbody: ReportBody,
```

Comment:
```suggestion
    pub reportbody: ReportBody,
```

Is this supposed to be private?

Reply:
I don't think so. That was a mistake.